snmp/assets/dashboards/interface_performance.json

```
@@ -0,0 +1,521 @@
+        "content": "Current State",
+        "background_color": "orange",
+        "font_size": "18",
+        "text_align": "center",
+        "show_tick": false,
+        "tick_pos": "50%",
+        "tick_edge": "left"
+      }
+    },
+    {
+      "id": 15,
+      "layout": { "x": 78, "y": 137, "width": 56, "height": 5 },
+      "definition": {
+        "type": "note",
+        "content": "User-defined State",
+        "background_color": "orange",
+        "font_size": "18",
+        "text_align": "center",
+        "show_tick": false,
+        "tick_pos": "50%",
+        "tick_edge": "left"
+      }
+    },
+    {
+      "id": 16,
+      "layout": { "x": 78, "y": 143, "width": 56, "height": 32 },
+      "definition": {
+        "title": "Interface Status",
+        "title_size": "16",
+        "title_align": "left",
+        "type": "query_table",
+        "requests": [
+          {
+            "aggregator": "avg",
+            "conditional_formats": [
+              {
+                "comparator": "<=",
+                "palette": "black_on_light_green",
+                "value": 1
+              },
+              {
+                "comparator": "<=",
+                "palette": "black_on_light_red",
+                "value": 2
+              },
+              {
+                "comparator": ">",
+                "palette": "black_on_light_yellow",
+                "value": 2
+              }
+            ],
+            "order": "desc",
+            "q": "avg:snmp.ifAdminStatus{$snmp_host,$interface,$snmp_device} by {snmp_device,snmp_host,interface}",
+            "alias": "admin status",
+            "limit": 50,
+            "cell_display_mode": ["bar"]
+          }
+        ],
+        "has_search_bar": "auto"
+      }
+    },
+    {
+      "id": 17,
+      "layout": { "x": 0, "y": 137, "width": 20, "height": 28 },
+      "definition": {
+        "type": "note",
+        "content": "To fully understand the current state of a port, we compare two different metrics, `snmp.AdminStatus` and `snmp.OperStatus`. \n\n**Admin Status** is defined by the SNMP manager. This is the state a port is supposed to be at configuration time. \n\nMeanwhile, the actual current state of the port is known as the **Operational Status**.",
+        "background_color": "orange",
+        "font_size": "14",
+        "text_align": "left",
+        "show_tick": false,
+        "tick_pos": "50%",
+        "tick_edge": "left"
+      }
+    },
+    {
+      "id": 18,
+      "layout": { "x": 135, "y": 143, "width": 23, "height": 28 },
+      "definition": {
+        "type": "note",
+        "content": "**Oper Status**\n\n```\n1 - up\n2 - down\n3 - testing\n4 - unknown\n5 - dormant\n6 - notPresent\n7 - lowerLayerDown\n```\n\n**Admin Status**\n```\n1 - up\n2 - down\n3 - testing\n```",
+        "background_color": "orange",
+        "font_size": "14",
+        "text_align": "left",
+        "show_tick": false,
+        "tick_pos": "50%",
+        "tick_edge": "left"
```

```suggestion
        "background_color": "gray",
        "font_size": "14",
        "text_align": "left",
        "show_tick": true,
        "tick_pos": "50%",
        "tick_edge": "left"
```

Same suggestion for note pointer and background color.